db/schema.rb

```diff
@@ -18,6 +18,7 @@
     t.datetime "created_at", precision: 6, null: false
     t.datetime "updated_at", precision: 6, null: false
     t.string "password_digest"
+    t.index ["email"], name: "index_users_on_email", unique: true
```

Comment:
sao k có migrate thay đổi mà schema lại thay đổi e?

Reply:
Cái này em có migrate rồi ạ, chắc do em sửa commit nhiều nó bị sót ở đâu, đó, em add index to users_email, trước cái add mật khẩu cơ ạ